docs/user_guide/backend_configuration.rst

```diff
@@ -139,6 +139,34 @@ Then we can use this configuration to write the NWB file:
     configure_and_write_nwbfile(nwbfile=nwbfile, backend_configuration=backend_configuration, nwbfile_path="output.nwb")
 
 
+Existing configuration
```

Comment:
I think this is the core feature that is added in this PR. This allow the user to get the current configuration, modify what they need to modify and then write to nwbfile. The feature is quite flexible in that way.

On top of that repack is a wrapper of this behavior that works for a sub-set of use cases with fewer lines.